pascal-zarrad/psh
# Feature/implement multi user

### Description
Fix many code style issues and source out some reuseable code parts.
Improve code basis.
Implement multi-user functionality which now enables a fully automated/unattended installation of psh for a specific user.

### Issue
This PR addresses the following issues:
 - pascal-zarrad/psh #19  | Make psh multi user ready

### Test Scenarios
What is the expected behavior that is expected by the changes of the PR?
 - Run psh with a speicifc user - should work as always without any errors with any param.
 - Run psh (not as root) and define a user. It should throw an error.
- Run psh as root and define a specific user - it should install psh for that user. The installed files should be the same as in the current user installation.
